src/earn/EarnActivePool.tsx

```diff
@@ -0,0 +1,150 @@
+import React from 'react'
+import { useTranslation } from 'react-i18next'
+import { StyleSheet, Text, View } from 'react-native'
+import { useSelector } from 'react-redux'
+import Button, { BtnSizes, BtnTypes } from 'src/components/Button'
+import SkeletonPlaceholder from 'src/components/SkeletonPlaceholder'
+import TokenDisplay from 'src/components/TokenDisplay'
+import { useAavePoolInfo, useAavePoolUserBalance } from 'src/earn/hooks'
+import UpwardGraph from 'src/icons/UpwardGraph'
+import Colors from 'src/styles/colors'
+import { typeScale } from 'src/styles/fonts'
+import { Spacing } from 'src/styles/styles'
+import { useTokenInfo } from 'src/tokens/hooks'
+import networkConfig from 'src/web3/networkConfig'
+import { walletAddressSelector } from 'src/web3/selectors'
+import { Address } from 'viem'
+
+function PoolDetailsLoading() {
+  return (
+    <View>
+      <SkeletonPlaceholder backgroundColor={Colors.gray2} highlightColor={Colors.white}>
+        <View
+          style={{
+            ...typeScale.labelXSmall,
+            width: 64,
+            borderRadius: 100,
+          }}
+        />
+      </SkeletonPlaceholder>
+    </View>
+  )
+}
+
+export default function EarnActivePool() {
+  const { t } = useTranslation()
+  const token = useTokenInfo(networkConfig.arbUsdcTokenId)
+  const walletAddress = useSelector(walletAddressSelector)
+  const asyncPoolInfo = useAavePoolInfo(token?.address as Address)
+  const asyncPoolUserInfo = useAavePoolUserBalance({
+    assetAddress: token?.address as Address,
+    walletAddress: walletAddress as Address,
+  })
+
+  return (
+    <View style={styles.card} testID="EarnActivePool">
+      <View style={styles.container}>
+        <Text style={styles.title}>{t('earnFlow.activePools.title')}</Text>
+        <View>
+          <View style={styles.row}>
+            <Text style={styles.totalValueText}>{t('earnFlow.activePools.totalValue')}</Text>
+            {asyncPoolUserInfo.loading && <PoolDetailsLoading />}
+            {asyncPoolUserInfo.result && (
+              <TokenDisplay
+                amount={asyncPoolUserInfo.result.balanceInDecimal}
+                showLocalAmount={false}
+                testID={`${networkConfig.arbUsdcTokenId}-totalAmount`}
+                tokenId={networkConfig.arbUsdcTokenId}
+              />
+            )}
+          </View>
+          <View style={styles.row}>
+            {asyncPoolInfo.loading && <PoolDetailsLoading />}
+            {asyncPoolInfo.result && (
+              <View style={styles.apyContainer}>
+                <Text style={styles.apyText}>
+                  {t('earnFlow.activePools.apy', {
+                    apy: (asyncPoolInfo.result.apy * 100).toFixed(2) ?? '',
```

Comment:
we've ran into "percentage naming" issues before: where we name something "percentage" (e.g., APY) when the value is in fact a proportion (i.e., the "decimal value") and then end up passing the value to an API or showing to the user or etc when the expected value is a percentage (or vice versa).  Can we get ahead of it here somehow?

I realize the naming I'm commenting on might predate this PR.  How about `result.apyProporion` (don't think to hard about Annual Percentage Yield Proportion :laughing: ) and `apy` (for the actual percentage)